pkg/plugins/scorer/kvcache_aware.go

```diff
@@ -18,44 +20,58 @@ import (
 const (
 	// KvCacheAwareScorerType is the type of the KvCacheAwareScorer
 	KvCacheAwareScorerType = "kvcache-aware-scorer"
-
-	kvCacheRedisEnvVar     = "KVCACHE_INDEXER_REDIS_ADDR"
-	huggingFaceTokenEnvVar = "HF_TOKEN"
 )
 
+type kvCacheAwareScorerParameters struct {
+	RedisAddres string `json:"redisAddress"`
+	HfToken     string `json:"hfToken"`
```

Comment:
don't you think this is problematic?
putting personal access token or connection string that includes password, in a configuration file?

Reply:
I revered the code here. These two fields are again fetched from environment variables